packages/babel-core/src/config/files/configuration.js

```diff
@@ -2,6 +2,7 @@
 
 import buildDebug from "debug";
 import path from "path";
+import codeFrame from "@babel/code-frame";
```

Comment:
The default import from this is marked deprecated: https://github.com/babel/babel/blob/9ad8b2bb2c239505a874003166fdb1ca9189feee/packages/babel-code-frame/src/index.js#L183